button.css

```diff
@@ -0,0 +1,82 @@
+.btn {
+  padding: 1rem 1.2rem;
+  font-weight: bold;
+  margin: 1.2rem;
+  cursor: pointer;
+  background: none;
+  outline: inherit;
+  color: var(--white);
+  border: none;
+}
+.btn:active {
+  border: 2px solid;
+  border-color: var(--alertblue);
+}
+.btn-primary {
+  background-color: var(--primary);
+}
+.font-sm {
+  font-size: 1rem;
+}
```

Comment:
This class should be in the utility stylesheet since it will be used across many components and not just this one